packages/cli/src/commands/experimental/templates/realtime.ts.template

```diff
@@ -0,0 +1,15 @@
+import { createPubSub, InMemoryLiveQueryStore } from '@redwoodjs/graphql-server'
+
+import type { NewMessageChannel } from 'src/subscriptions/newMessage/newMessage'
+
+export const liveQueryStore = new InMemoryLiveQueryStore()
+
+export type OldMessageChannel = {
+  oldMessage: [payload: { name: string }]
+}
+
+export type PubSubChannels = NewMessageChannel<string> & OldMessageChannel
```

Comment:
I don't think this type is being updated when you run the generator for a subscription. This results in a typescript error later on in the service file.

In the service we have a line like so:

```ts
context.pubSub.publish(`Countup:${id}`, { from, body })
```
which raises the following error:
```
Argument of type '`Countup:${string}`' is not assignable to parameter of type '`newMessage:${string}` | "oldMessage"'.ts(2345)
```

---
On a similar, but unimportant, note should we be specifying the type for `publish` so that in the example `send` function of `rooms/rooms.ts` you have:
```ts
context.pubSub.publish<`newMessage:${string}`>(`newMessage:${roomId}`, {
  from,
  body,
})
```
instead of:
```ts
context.pubSub.publish(`newMessage:${roomId}`, { from, body })
```

I have to admit that I don't understand the `OldMessageChannel` yet.

Reply:
The `OldMessage` should not have been added to the template. It was my test to see how one would `&` types together from each exported subscription.

The generator tries to explain how to do this:

```
// Be sure to import this type and then in `api/src/lib/realtime.ts` extend PubSubChannels:
// export type PubSubChannels = ExistingChannel<string> & Publish${typeName}Channel<string>
```

But, it is clunky. 

It's an item I want to get feedback from devs on how best to do.

So, I'll hold off on any changes (except removing the OldMessage) until I get more feedback.